PKHeX/Legality/Checks.cs

```diff
@@ -929,7 +973,35 @@ private CheckResult verifyEncounterG3Transfer()
         private CheckResult verifyEncounterG4Transfer()
         {
             CheckResult Gen4Result = null;
-            
+            CheckResult Gen4WildResult = null;
+            object WildEncounter = null;
```

Comment:
I thought first it was not possile because EncounterMatch is an object that can be many types, but WildEncounter only store the result of getWildEncounters that have EncounterSlot[] type, i change it to that type